docfx/api-override/Meadow.Foundation.Grove/Meadow.Foundation.Grove.Sensors.Switches.SwitchP.md

```diff
@@ -3,10 +3,10 @@ uid: Meadow.Foundation.Grove.Sensors.Switches.SwitchP
 remarks: *content
 ---
 
-| Switch(P) | |
+| SwitchP | |
```

Comment:
Did we need a hack for the real name on all these component variations ("Switch(P)")?